paper-spinner-behavior.html

```diff
@@ -0,0 +1,104 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<script>
+
+  /** @polymerBehavior */
+  Polymer.PaperSpinnerBehavior = {
+
+    listeners: {
+      'animationend': 'reset',
+      'webkitAnimationEnd': 'reset'
+    },
+
+    properties: {
+      /**
+       * Displays the spinner.
+       *
+       * @attribute active
```

Comment:
These annotations are mostly unnecessary now. The compiler and Hydrolysis are both sufficiently robust that they can infer the values from the property declaration. The only time we need them now is if the documentation should be different from the declaration for some reason.